src/aura/HH_AutoComplete/HH_AutoCompleteController.js

```diff
@@ -0,0 +1,23 @@
+({
+    handleNewContact: function(component, event, helper) {
+        var con = {'sobjectType': 'Contact'};
+        var autocomplete = component.find('autocomplete');
+        var name = autocomplete.get('v.displayValue');
+        var list = name.split(' ');
+        if (list.length == 0) {
```

Comment:
I wanted to protect for name being an empty string.  not sure what split would return then.